### ISSUE TITLE
Accessibility - Main Menu Steps - VoiceOver Reads Steps but Topics and Other Step Disappear from Menu

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#80]
I can now activate the menu items with VoiceOver but when I open a given Step and the look at the individual Topics under that Step all the other Steps are visible gone.

**Comment by jfc3-dol** [comment]
Same thing happens with Android too.

**Comment by adriennerlong** [comment]
From ICF: Intentional.  Other steps become hidden to not overwhelm the user.

**Comment by jfc3-dol** [comment]
But I should be able to get to the other Steps and be able to expand them too. I shouldn't only see the one set of expanded steps.

When attempting to scroll the list of Steps the content on the screen behind the drop-down main menu scrolls. That is if it has more than a screens worth of content instead of the items on the navigation menu.

**Comment by jfc3-dol** [comment]
TalkBack reads the Topics within a given Step and moves down the screen to show the ones that are not visually there.

For some reason, TalkBack continues to on to read the main content of the screen behind the main menu when it shouldn't.

Going to remove the Android label for this issue and open another one for TalkBack reading past the end of the main menu content.

**Comment by jfc3-dol** [comment]
While testing iOS version after opening the list of Topics under Step 2 in the main menu VoiceOver starts reading items on the page and not in the navigation.
